lib/reaction.cc

```diff
@@ -6,6 +6,10 @@
  *   Christian Menard
  */
 
+#include <utility>
```

Comment:
Duplicated include. Also, is this include needed? I didn't see any changes that would need the additional header.